Libplanet/Net/Swarm.cs

```diff
@@ -87,6 +91,7 @@ public class Swarm : ICollection<Peer>, IDisposable
             PrivateKey privateKey,
             int appProtocolVersion,
             TimeSpan dialTimeout,
+            TimeSpan linger,
```

Comment:
A naming suggestion: how about `gracePeriod`?

Reply:
`gracePeriod` is also good, but I think `linger` is general enough to explain this behavior.